README.md

```diff
@@ -1 +1,287 @@
-# ballerina-gke-deployment
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/ballerina-gke-deployment.svg?branch=master)](https://travis-ci.org/ballerina-guides/ballerina-gke-deployment)
+
+# Ballerina Deployment with Google Cloud Platoform Kubernetes Engine
+
+[Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/) is a powerful cluster manager and orchestration system for running your Docker containers
+
+> In this guide you will learn about building a Ballerina service and deploying it on Google Kubernetes Enginer (GKE).
+
+The following are the sections available in this guide.
```

Comment:
Are we not covering observability here?

Reply:
No, we are focusing on a simple guide on a remote deployment on Google Kubernetes Engine. Also I don't think it makes sense set up the observability stuff similarly to how we did locally. 
May be we can do it properly as an improvement later.